### PR TITLE
VZ-9862 Check if workloads are ready

### PR DESCRIPTION
**Pull request by pfmackin**
Check if the Helm workloads are ready.  This includes deployment, daemonset, statefulset.

### CODE REVIEW

File: pkg/helm/helmcli.go [hunk 1]
```diff
@@ -70,6 +70,24 @@ func SetDefaultLoadChartFunction() {
...
+func GetRelease(log vzlog.VerrazzanoLogger, releaseName string, namespace string) (*release.Release, error) {
+	settings := cli.New()
+	settings.SetNamespace(namespace)
+	actionConfig, err := actionConfigFn(log, settings, namespace)
+	if err != nil {
+		return nil, err
+	}
+
+	client := action.NewGet(actionConfig)
+	rel, err := client.Run(releaseName)
```
Comment (fredtibbitts) [inline pkg/helm/helmcli.go:83]: Minor, you can simplify this and just `return client.Run(releaseName)`

**Review by fredtibbitts — DISMISSED** [review]
A few minor comments, and several modules need imports formatted.

File: module-operator/controllers/module/handlers/common/workloads_ready.go [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+	"github.com/verrazzano/verrazzano-modules/pkg/k8s/readiness"
+	v1 "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const helmKey = " meta.helm.sh/release-name"
+
+func CheckWorkLoadsReady(ctx handlerspi.HandlerContext, releaseName string, namespace string) (bool, error) {
+	// return true ifall the workloads are ready
```
[the remark this answers is not in the document]
Comment (pfmackin) [reply module-operator/controllers/module/handlers/common/workloads_ready.go]: fixed

File: module-operator/controllers/module/handlers/common/base_handler.go [hunk 3]
```diff
@@ -106,6 +106,7 @@ func (h BaseHandler) CheckReleaseDeployedAndReady(ctx handlerspi.HandlerContext)
 		return false, util.NewRequeueWithShortDelay(), nil
 	}
 
-	// TODO check if release is ready (check deployments)
-	return true, ctrl.Result{}, err
+	// Wait until all the workload pods are ready
```
[the remark this answers is not in the document]
Comment (pfmackin) [reply module-operator/controllers/module/handlers/common/base_handler.go]: fixed

File: module-operator/controllers/module/handlers/common/base_handler.go [hunk 4]
```diff
@@ -96,7 +96,7 @@ func (h BaseHandler) CheckReleaseDeployedAndReady(ctx handlerspi.HandlerContext)
 		ctx.Log.Debugf("IsReady() dry run for %s", h.HelmRelease.Name)
 		return true, ctrl.Result{}, nil
 	}
-
+	// Wait until Helm says release is deployed
```
[the remark this answers is not in the document]
Comment (pfmackin) [reply module-operator/controllers/module/handlers/common/base_handler.go]: fixed

**Review by fredtibbitts — APPROVED** [review]

**Pull request merged**